### PR TITLE
normalize artifact dirs to write to separate places

### PR DESCRIPTION
**Pull request by jcantrill**
This PR attempts to normalize the artifact directories so artifacts are separated by test

### CODE REVIEW

File: test/helpers/clusterlogging.go [hunk 1]
```diff
@@ -51,6 +51,11 @@ func NewClusterLogging(componentTypes ...LogComponentType) *cl.ClusterLogging {
 					NodeCount:        1,
 					RedundancyPolicy: elasticsearch.ZeroRedundancy,
 				},
+				RetentionPolicy: &cl.RetentionPoliciesSpec{
```
Comment (ewolinetz) [inline test/helpers/clusterlogging.go]: looking at the code it looks like we just check if the fields are `nil` so we could possibly just have:
```
RetentionPolicy: &cl.RetentionPoliciesSpec{
  App: &cl.RetentionPolicySpec{},
  Infra: &cl.RetentionPolicySpec{},
  Audit: &cl.RetentionPolicySpec{},
}
```

Comment (ewolinetz) [reply test/helpers/clusterlogging.go]: https://github.com/openshift/cluster-logging-operator/pull/426

Comment (ewolinetz) [reply test/helpers/clusterlogging.go]: maybe not.. `"Error occured while getting hot phase age for App log source. err: age pattern mismatch"`

Comment (ewolinetz) [inline test/helpers/clusterlogging.go]: https://github.com/openshift/cluster-logging-operator/pull/426

Comment (jcantrill) [reply test/helpers/clusterlogging.go]: reverted

**Pull request merged**